### DISCUSSION TITLE
Multiple browser versions

### DISCUSSION THREAD

**Discussion by oskarm93** [SeleniumHQ/docker-selenium#2446] [Q&A] [answered]
I am deploying Selenium Grid with the Helm chart. There is very little information about how to configure a different version of browser drivers or even multiple browser versions. Testers want to run against Chrome version 75, 90, 100 and latest.

Do we need separate nodes for each version? 
Can we have multiple versions on the same node? 
How to configure this? 
Is this even supported by the Helm chart?

If I try to override Helm chart values with another node tag, it does not even register that node in the hub...
```
chromeNode:
  imageTag: "90.0"
```

**Comment by VietND96** [comment]
Hi, yes, for different browser versions, we need to separate node.
In Helm chart, there is limitation that number of nodes and templates are fixed.
If this is a use case that people prefer to, we will try to support this soon

**Comment by VietND96** [comment, marked as answer]
The implementation is tracked via https://github.com/SeleniumHQ/docker-selenium/issues/2456
